src/Microsoft.TestPlatform.CrossPlatEngine/AttachmentsProcessing/TestRunAttachmentsProcessingManager.cs

```diff
@@ -8,63 +8,64 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Xml;
 using Microsoft.VisualStudio.TestPlatform.Common.Telemetry;
 using Microsoft.VisualStudio.TestPlatform.CoreUtilities.Tracing.Interfaces;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
-using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Engine;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Utilities;
 
 namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.TestRunAttachmentsProcessing
 {
     /// <summary>
     /// Orchestrates test run attachments processing operations.
     /// </summary>
-    public class TestRunAttachmentsProcessingManager : ITestRunAttachmentsProcessingManager
+    internal class TestRunAttachmentsProcessingManager : ITestRunAttachmentsProcessingManager
```

Comment:
Like the above, it's internal implementation not used around because attachment processor is new and until this PR is not possible for users register an attachment processor, but I agree that we can start to cleanup public api with more numbers in follow-up PRs